### PR TITLE
Quick bug fix and update local net ACLs

### PR DESCRIPTION
**Pull request by digitalrane**
- Updates ACLs to include 172.x local subnets, and IPv6 local and auto subnets.
- Fixed a bug with health checking based on urlbase not having / prepended
- These should be separate PRs, I know.

### CODE REVIEW

File: src/tests/functional/test_deploy.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 pytestmark = pytest.mark.asyncio
 
 series = ['xenial', 'bionic']
-juju_repository = os.getenv('JUJU_REPOSITORY', '.').rstrip('/')
+juju_repository = os.getenv('juju_repository', '.').rstrip('/')
```
Comment (chris-sanders) [inline src/tests/functional/test_deploy.py]: What's this for?

**Review by chris-sanders — COMMENTED** [review]
One question

File: src/tests/functional/test_deploy.py (hunk 1 above)
Comment (digitalrane) [reply src/tests/functional/test_deploy.py]: Apparently just for breaking functional testing.
```
:undo
:wq
```
Fixed

**Comment by chris-sanders** [comment]
lgtm

**Pull request merged**